packages/tempest/src/Text.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import * as React from 'react'
```

Comment:
Looks like some of these need to be switched back to the import `React from 'react'` syntax?